devtools/maven/src/main/java/io/quarkus/maven/ListExtensionsMojo.java

```diff
@@ -20,22 +20,28 @@
     /**
      * List all extensions or just the installable.
      */
-    @Parameter(property = "quarkus.extension.all", alias = "quarkus.extension.all", defaultValue = "true")
+    @Parameter(property = "all", defaultValue = "true")
     protected boolean all;
 
     /**
      * Select the output format among 'name' (display the name only), 'concise' (display name and description) and 'full'
      * (concise format and version related columns).
      */
-    @Parameter(property = "quarkus.extension.format", alias = "quarkus.extension.format", defaultValue = "concise")
+    @Parameter(property = "format", defaultValue = "concise")
```

Comment:
This is now inconsistent with the property you have defined for category. `quarkus.extension.format` .. 

we need to pick one (across all of the quarkus maven plugin mojos).. long form or short form. @aloubyansky , I don't have the history here, and I know you've messed with this more than most...